bundles/org.eclipse.ui.ide/src/org/eclipse/ui/internal/views/markers/MarkerEntry.java

```diff
@@ -385,6 +388,7 @@ Map<String, Object> getCache() {
 	@Override
 	void clearCache() {
```

Comment:
`clearCache()` seem to be called on **every** marker. Does it make sense to have a dedicated method for the static cache cleanup? Just to avoid the map is recreated 100_000 times, also it sounds strange to cleanup static data in non static context.